src/urbit/app/chess.hoon

```diff
@@ -853,10 +813,12 @@
       =/  game-state  (~(got by games) u.game-id)
       =/  fen  (position-to-fen position.game-state)
       =/  cards  ^-  (list card)
-        :~  :*  %give  %fact  ~[/game/(scot %da u.game-id)/updates]
-                %chess-update  !>([%position u.game-id fen special-draw-available.game-state])
-            ==
-        ==
+         %+  turn
+           moves.game.game-state
+         |=  move=[move=chess-move fen=chess-fen san=chess-san]
+         :*  %give  %fact   ~[/game/(scot %da u.game-id)/updates]
+             %chess-update  !>([%position u.game-id fen.move | san.move])
```

Comment:
We're ignoring `special-draw-available.game-state` here (always returning the special draw state as false). Safe for now to just return whatever the current state is for every update, though it's not optimal for the correct function of interfaces.

Reply:
I replaced `special-draw-available` with `|` because of the auto-claim setting. If player has auto-claim on, then receives a `%position` update where special draw is `&`, will the frontend end that game as it's being populated in the frontend? @rovmug-ticfyn